jellyfin_mpv_shim/rich_presence.py

```diff
@@ -17,8 +23,14 @@ def send_presence(
     duration: float = None,
     playing: bool = False,
     syncplay_group: str = None,
+    artwork_url: str = None,
 ):
-    small_image = "play-dark3" if playing else None
+    small_image = (
+        "https://cdn.discordapp.com/app-assets/463097721130188830/493061639994867714.png"
+        if playing
+        else "https://cdn.discordapp.com/app-assets/463097721130188830/493061640296595456.png"
+    )
```

Comment:
I can upload these so we're not hard-coding the URL in the source code.

Reply:
Yes, that would be best